framework/bin/run_mutation.pl

```diff
@@ -287,7 +294,11 @@ sub _run_mutation {
 
     # Create mutation definitions (mml file)
     my $mml_dir = "$TMP_DIR/.mml";
-    system("$UTIL_DIR/create_mml.pl -p $PID -c $TARGET_CLASSES_DIR/$bid.src -o $mml_dir -b $bid");
+    if (defined $MUTATE_CLASSES) {
```

Comment:
See comment above about desirable behavior if both flags (-i and -A) are provided.